### DISCUSSION TITLE
How to prevent user from drawing new links when locked

### DISCUSSION THREAD

**Discussion by JeremyGEA** [Blazor-Diagrams/Blazor.Diagrams#250] [Q&A] [answered]
I'm using the node and link Locked property as a method of determining when a user is in "Edit" (unlocked) mode and "View" (locked) mode. When in "View" mode, I don't want them to be able to click on a port and create a new link. How do would I prevent this?

**Comment by zHaytam** [comment, marked as answer]
You can lock the port as well

**Reply by JeremyGEA** [reply]
Thanks! That was it.
